bundles/binding/org.openhab.binding.zwave/database/products.xml

```diff
@@ -1879,6 +1879,18 @@
 			<Label lang="en">Wall Switch Set for Busch Jaeger</Label>
 			<ConfigFile>zwaveme/zme_05457.xml</ConfigFile>
 		</Product>
+		<Product>
+			<Reference>
+				<Type>0024</Type>
+				<Id>0001</Id>
+			</Reference>
+			<Model>ZME_FT</Model>
+			<Label lang="en">Floor Thermostat</Label>
+			<ConfigFile>zwaveme/zme_ft11.xml</ConfigFile>
+			<ConfigFile VersionMax="1.2">zwaveme/zme_ftll.xml</ConfigFile> <!-- VersionMin="1.1" -->
+			<ConfigFile VersionMax="1.3">zwaveme/zme_ftl3.xml</ConfigFile> <!-- VersionMin="1.3" -->
```

Comment:
Hmmm - this notation might not be a good idea. I think that for the middle one you should use min and max, and for the 1.4 version, it should be versionMin, not max...  Otherwise I don't think the version comparison will work properly...